requirements.txt

```diff
@@ -10,3 +10,4 @@ git+https://github.com/apluslms/a-plus-client.git@v1.1.0#egg=a_plus_client
 python-memcached >= 1.59, < 2
 git+https://github.com/apluslms/greedy-string-tiling.git@v0.12.0
 django-debug-toolbar >= 3.2
+aplus-auth~=0.2.0
```

Comment:
This file already used the cleaner coding style that uses whitespace around the operators.

Reply:
Fixed